vector/src/test/java/im/vector/app/features/settings/devices/v2/ParseDeviceUserAgentUseCaseTest.kt

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.settings.devices.v2
+
+import im.vector.app.features.settings.devices.v2.list.DeviceType
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.Test
+
+private val A_USER_AGENT_LIST_FOR_ANDROID = listOf(
```

Comment:
I think it would be good to test some weird user agents since they can be fully customised by other applications or browser extensions like empty string, `curl/7.55.1`, `some random string` etc

Reply:
Right, added. Thanks.